java/code/webapp/WEB-INF/struts-config.xml

```diff
@@ -3232,6 +3232,8 @@
                  path="/WEB-INF/pages/systems/sdc/history_event.jsp"/>
         <forward name="continue"
                  path="/systems/details/history/History.do"/>
+        <forward name="confirm"
```

Comment:
Agreed. In the beginning I thought it would need to be confirm, since the forward in the `SPMigrationAction` is also called confirm, but that isn't the case.
So I will change it to `spmigration`